index.html

```diff
@@ -259,6 +259,66 @@
         width: 100%;
       }
     }
+
+@media screen and (prefers-color-scheme: dark) {
+      body {
+          background: #333 !important;
```

Comment:
Your editor has a curious way of setting the indentation level. How does it work?

Regardless of the answer, please indent by steps of 2 spaces, like the rest of the code. I personally prefer deeper indentation like you have done here, but consistency is important.

**Edit:** I now realize what is going on. You were using regular 4-space indentation, but line 263 is missing the two leading spaces that would match it with its closing brace.

Reply:
I changed my tabstop to 2 to better conform.  

applied the suggest filter (thanks)

and indeed, a little more dark mode to the dark mode

Reply:
Thanks. I will look at it again tomorrow.